package.json

```diff
@@ -42,6 +42,7 @@
     "open-color": "1.9.1",
     "pako": "1.0.11",
     "perfect-freehand": "1.0.16",
+    "pica": "^7.1.1",
```

Comment:
@7flash also can you lock the version ? as we are doing that for all deps so it has same build (lock file), you will need to run run `yarn` again to update lock file